modules/core/24-host/channel_keys.go

```diff
@@ -52,3 +54,8 @@ func ChannelRestoreKey(portID, channelID string) []byte {
 func channelPath(portID, channelID string) string {
 	return fmt.Sprintf("%s/%s/%s/%s", KeyPortPrefix, portID, KeyChannelPrefix, channelID)
 }
+
+// ErrorPath stores the ErrorReceipt in the case that a chain does not accept the proposed upgrade
+func ErrorPath(portID, channelID string) string {
+	return fmt.Sprintf("%s/%s/%s/%s/%s/%s", KeyChannelUpgradePrefix, KeyPortPrefix, portID, KeyChannelPrefix, channelID, KeyUpgradeError)
```

Comment:
I think we can simplify this like below:

```suggestion
	return fmt.Sprintf("%s/%s/%s", KeyChannelUpgradePrefix, KeyUpgradeErrorPrefix, channelPath(portID, channelID))
```

Reply:
good call! i'll update

Reply:
@damiannolan this change isn't equivalent to the current string, but I believe this new format is correct. The existing string has `upgradeError` at the end, when we actually want it as `channelUpgrades/upgradeError/...`